proxy/contracts/mainnet/CommunityPool.sol

```diff
@@ -212,10 +248,33 @@ contract CommunityPool is Twin, ICommunityPool {
     function checkUserBalance(bytes32 schainHash, address receiver) external view override returns (bool) {
         return activeUsers[receiver][schainHash] && _balanceIsSufficient(schainHash, receiver, 0);
     }
+
+    /**
+     * @dev Checks whether passed amount is enough to recharge user wallet with current basefee.
+     */
+    function getRecommendedRechargeAmount(
+        bytes32 schainHash,
+        address receiver
+    )
+        external
+        view
+        override
+        returns (uint256)
+    {
+        if (_multiplyOnAdaptedBaseFee(minTransactionGas)  <= _userWallets[receiver][schainHash]) {
```

Comment:
You may calculate it once and remember in variable.

Reply:
Yeap agree